files/en-us/web/api/csskeywordvalue/value/index.html

```diff
@@ -24,6 +24,13 @@ <h3 id="Value">Value</h3>
 
 <p>A {{domxref('USVString')}}.</p>
 
+<h3 id="Exceptions">Exceptions</h3>
+
+<dl>
+	<dt><code>TypeError</code></dt>
+  <dd>If the <code>value</code> property is an empty {{jsxref('String')}} when being set.</dd>
```

Comment:
```suggestion
  <dt><code>TypeError</code></dt>
  <dd>If the <code>value</code> property is an empty {{jsxref('String')}} when being set.</dd>
```
Space instead of tabs I guess